tests/unit/test_z_cmdline.py

```diff
@@ -789,6 +789,32 @@ def test_notest_setup_py_error(initproj, cmd):
     assert re.search("ERROR:.*InvocationError", result.out)
 
 
+def test_devenv(initproj, cmd):
```

Comment:
also, add a test that triggers the ``"--devenv requires only a single -e"`` part 👍